libqtile/widget/__init__.py

```diff
@@ -38,6 +38,7 @@ def safe_import(module_name, class_name):
 
 safe_import("backlight", "Backlight")
 safe_import("battery", ["Battery", "BatteryIcon"])
+safe_import("battery_monitor", ["BatteryMonitor", "BatteryMonitorIcon"])
```

Comment:
Why put this in a different file? Can we just adjust the battery to add this functionality via your FreeBSD implementation?

Reply:
I put it in a separate file because it was simpler while working on it. If or when the Linux integration happens, I was planning on putting it back in the old file. I believe I explained it some in the PR or the issue that spawned the PR.